scripts/schemaGenerator.py

```
@@ -19,8 +19,23 @@
     apply_json_diff,
     get_formatted_json,
     is_old_format,
+    should_add_dynamic_config_pattern,
+    should_add_env_pattern,
+    is_generic_fallback,
+    has_explicit_dynamic_config_pattern,
+    has_explicit_env_pattern,
+    DEFAULT_GENERIC_FALLBACK,
+    GENERIC_FALLBACK_PATTERN,
+    DYNAMIC_CONFIG_REGEX_PATTERN,
```

Ruff (0.12.2)

`utils.DYNAMIC_CONFIG_REGEX_PATTERN` imported but unused

Remove unused import

(F401)